### ISSUE TITLE
FrSky S8R Blue LED won't turn On

### ISSUE THREAD

**Issue by szempruw** [opentx/opentx#8493]
I am a "normal" FrSky User. I updated old X9D Tx and older X9D+ (pre 2019) using Companion 2.3.11. I read about need to update FrSky receiver software to 2.1.x Software. I updated internal module in Tx to XJT 2.1 off the FrSky website. Also updated all my Rx's to the latest 2.1 .x software and specifically for the S8R Rx using SXR_ACCST_2.1.0_FCC.frk.  

The update was flashed successfully using the Tx FIRMWARE directory, with firmware loaded on the SD Card and flashed to external device connected to the Rx Smart port.

I bound the Rx using XJT D16, 1-16 Ch and Telemetry On and then used the scripts copied into the SD Card to calibrate the receiver. Calibration successful. (Thumbs up) Using the S8R setup lua script, I chose orientation and enabled both SxR function and Quick Mode. I left all other values alone. 

I installed Rx into a plane. I tried the 3-pos switch on Channel 12 to attempt the Self Check routine. The **Blue LED did not come on.** I next tried to press the F/S  button on the Rx and there was **no Blue LED** or servo twitch as I expected. I moved the other joystick function as indicated and then the throttle servo. The plane's functions responded correctly to movement as expected. The assigned Channel 10 switch turns the QM off to Stabilize to Auto-level as expected. The software seems to work, just no confirmation or indication of Self Check routine working and no temporary Blue LED when first turning the Rx on 

I have two S8R's and both will not turn Blue LED on. I only get the yellow, red and green lights as expected, green to indicate bind ok. 

I reference RCDIY  Oscar Liang video for correct setup of an S8R as I know it as a "normal" user. http://rcdiy.ca/sxr-setup-s6r-s8r/

I don't know if this Blue Light issue is FrSky Rx 2.1.x  or Lua Script or OpenTx software related. It just doesn't seem to work as expected.

Kindly, WTS

**Comment by szempruw** [comment]
I ensured that the order of setup was done correctly. Bind, Calibrate Rx using Calibration Lua, Use setup Lua to choose wing type and mounting type (in my case Horizontal Reverse), and ensure SxR function and QM is enabled. My throttle was reversed so Thr forward was -100%. The 3-pos switch activated the Blue LED. You have to move sticks quickly or it will time out and go green and then must repeat.

At least for now, all the issues I had were related to not having at least the switch invoke the self check routine. Ensuring that the Thr is at or below -100% was key. Again, the self check had to be repeated a few times to ensure the joysticks were moved before the witch occurred. If not, I would only get, for example, partial elevator movement.

Within RCDYI website written words, there was a comment made, that "depending on the software version used, the blue LED may not come on when Rx is powered up."
SxR – Stabilizer Gyro (S6R, S8R) – RCdiy
rcdiy.ca

Bottom line, the Rx Blue LED does not come on temporarily when in Quick Mode and powered up as was historically expected. The F/S button press does not activate self-check, Only the switch toggle. This little detail was changed in the new Rx update. I have one plane's complete functionality checked out with S8R 2.1.x
